onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -1009,6 +1036,42 @@ public void onClick(DialogInterface dialog, int which) {
                             new DialogInterface.OnClickListener() {
                                 @Override
                                 public void onClick(DialogInterface dialog, int which) {
+                                    createZipCodeDialog().show();
+                                }
+                            }
+                    );
+            return builder.create();
+        }
+
+        private Dialog createZipCodeDialog() {
+            Drawable icon = getResources().getDrawable(android.R.drawable.ic_dialog_map);
+            DrawableCompat.setTint(icon, getResources().getColor(R.color.theme_primary));
+            final EditText zipInput = new EditText(this.getContext());
+            zipInput.setInputType(InputType.TYPE_CLASS_NUMBER);
+
+            final AlertDialog.Builder builder = new AlertDialog.Builder(getActivity())
+                    .setTitle(R.string.main_ziplocation_title)
+                    .setIcon(icon)
+                    .setCancelable(false)
+                    .setMessage(R.string.main_ziplocation)
+                    .setView(zipInput)
```

Comment:
You can actually use the ZIP code preference dialog as guidance for what the padding should be:

![image](https://cloud.githubusercontent.com/assets/928045/16155778/f231c350-347e-11e6-85ad-184ed04f6de4.png)